pybliometrics/scopus/superclasses/base.py

```diff
@@ -154,3 +165,42 @@ def _check_file_age(self):
         refresh = True
         mod_ts = None
     return refresh, mod_ts
+
+def _get_references(res: dict) -> list:
+    """Get references list in responce"""
+    return res.get('abstracts-retrieval-response', {}).get('references', {}).get('reference', [])
```

Comment:
Can be replaced by `chained_get` from the utils module. No need for its own function.